src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -32,43 +38,43 @@ abstract contract StrategyManagerStorage is IStrategyManager {
     // maximum length of dynamic arrays in `stakerStrategyList` mapping, for sanity's sake
     uint8 internal constant MAX_STAKER_STRATEGY_LIST_LENGTH = 32;
 
-    // system contracts
-    IDelegationManager public immutable delegation;
-    IEigenPodManager public immutable eigenPodManager;
-    ISlasher public immutable slasher;
-
     /// @notice Permissioned role, which can be changed by the contract owner. Has the ability to edit the strategy whitelist
     address public strategyWhitelister;
 
-    /**
-     * @notice Minimum delay enforced by this contract for completing queued withdrawals. Measured in blocks, and adjustable by this contract's owner,
-     * up to a maximum of `MAX_WITHDRAWAL_DELAY_BLOCKS`. Minimum value is 0 (i.e. no delay enforced).
-     * @dev Note that the withdrawal delay is not enforced on withdrawals of 'beaconChainETH', as the EigenPods have their own separate delay mechanic
-     * and we want to avoid stacking multiple enforced delays onto a single withdrawal.
+    /*
+     * Reserved space previously used by the deprecated storage variable `withdrawalDelayBlocks.
+     * This variable was migrated to the DelegationManager instead.
      */
-    uint256 public withdrawalDelayBlocks;
-    // the number of 12-second blocks in one week (60 * 60 * 24 * 7 / 12 = 50,400)
-    uint256 public constant MAX_WITHDRAWAL_DELAY_BLOCKS = 50400;
+    // slither-disable-next-line incorrect-shift-in-assembly
+    uint256[1] internal _deprecatedStorage_withdrawalDelayBlocks;
 
     /// @notice Mapping: staker => Strategy => number of shares which they currently hold
     mapping(address => mapping(IStrategy => uint256)) public stakerStrategyShares;
     /// @notice Mapping: staker => array of strategies in which they have nonzero shares
     mapping(address => IStrategy[]) public stakerStrategyList;
-    /// @notice Mapping: hash of withdrawal inputs, aka 'withdrawalRoot' => whether the withdrawal is pending
+
+    /// @notice *Deprecated* mapping: hash of withdrawal inputs, aka 'withdrawalRoot' => whether the withdrawal is pending
+    /// @dev This mapping is preserved to allow the migration of withdrawals to the DelegationManager contract.
     mapping(bytes32 => bool) public withdrawalRootPending;
-    /// @notice Mapping: staker => cumulative number of queued withdrawals they have ever initiated. only increments (doesn't decrement)
-    mapping(address => uint256) public numWithdrawalsQueued;
+
+    /*
+     * Reserved space previously used by the deprecated mapping(address => uint256) cumulativeWithdrawalsQueued.
+     * This mapping tracked the cumulative number of queued withdrawals initiated by a staker.
+     * Withdrawals are now initiated in the DlegationManager, so the mapping has moved to that contract.
+     */
+    // slither-disable-next-line incorrect-shift-in-assembly
+    uint256[1] internal _deprecatedStorage_cumulativeWithdrawalsQueued;
+
```

Comment:
IMO deprecated storage values should just retain old naming, even if it's bad. People that actually care about what it used to point to will have an easier time cross-referencing

Reply:
I'm fine with that, can change back after responding to other comments

Reply:
switched back to old naming here. I think the comments are sufficient about deprecation, but please flag anything that's a bit lacking on clarity (or just make a quick commit to clarify the deprecation).

Reply:
Ah, I had meant keep the `_deprecated` prefix, just also keep the suffix as the exact prior name. The change here was that `numWithdrawalsQueued` -> `_deprecated_cumulativeWithdrawalsQueued`, and I was just advocating for `_deprecated_numWithdrawalsQueued`.

That said... I have no problem with dropping the `_deprecated` prefix and just leaving a comment. So, the changes are fine by me!